sdk/src/Core/Amazon.Runtime/Pipeline/HttpHandler/_bcl/HttpWebRequestFactory.cs

```diff
@@ -479,11 +479,18 @@ public virtual void ConfigureRequest(IRequestContext requestContext)
             {
                 if (_request.RequestUri.Scheme == Uri.UriSchemeHttp)
                 {
-                    _request.Proxy = requestContext.ClientConfig.GetHttpProxy();
+                    proxy = requestContext.ClientConfig.GetHttpProxy();
                 }
                 else if (_request.RequestUri.Scheme == Uri.UriSchemeHttps)
                 {
-                    _request.Proxy = requestContext.ClientConfig.GetHttpsProxy();
+                    proxy = requestContext.ClientConfig.GetHttpsProxy();
+                }
+
```

Comment:
This change was done because I found we broke debugging with fiddler because we are overriding the OS proxy by explicitly setting the `Proxy` property to null.